packages/create-svelte/tsconfig.json

```diff
@@ -3,11 +3,12 @@
 		"allowJs": true,
 		"checkJs": true,
 		"noEmit": true,
-		"noImplicitAny": true,
 		"target": "esnext",
 		"module": "esnext",
 		"moduleResolution": "node",
-		"allowSyntheticDefaultImports": true
+		"allowSyntheticDefaultImports": true,
+		"strict": true,
+		"skipLibCheck": true
```

Comment:
are we changing some typescript rules here in a patch version?

Reply:
No that's for the `create-svelte` project itself, not the template that's scaffolded